packages/core/src/modules/oob/OutOfBandApi.ts

```diff
@@ -75,6 +75,7 @@ export interface ReceiveOutOfBandInvitationConfig {
   autoAcceptConnection?: boolean
   reuseConnection?: boolean
   routing?: Routing
+  timeoutMs?: number
```

Comment:
Thinking whether we should give this another name, or whether we should set a timer for the overall action. If I set a timeoutMs I expect the whole action to have a specific timeout, but this only relates to the time we wait for the connection to be completed afterwards.

Also if autoAcceptConnection is false, this parameter won't do anything which may be a bit misleasing. 

What do you think?

Reply:
Yes, make sense, [FORCED PUSHED] changed the name to `acceptInvitationTimeoutMs`